client/constants/request-methods.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { APIUrl } from './constants';
+import { AxiosPublicInterface } from './interfaces';
+
+export enum REQUEST_METHOD {
+  GET = 'GET',
+  POST = 'POST',
+  PATCH = 'PATCH',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export const API_ENDPOINT = {
+  APPLICATION_ID: (applicationId: string) => `/applications/${applicationId}`,
+  APPLICATIONS: '/applications',
+  IN_PROGRESS: '/applications/in-progress',
+};
+
+export const AxiosPublic = ({ method, endpoint, token, data }: AxiosPublicInterface) => {
```

Comment:
It should be AxiosPrivate. This doesn;t require token, we can update this in onTokensChange in _app.tsx